packages/build-utils/src/index.ts

```diff
@@ -80,52 +80,10 @@ export {
 };
 
 export { EdgeFunction } from './edge-function';
-export {
-  detectBuilders,
-  detectOutputDirectory,
-  detectApiDirectory,
-  detectApiExtensions,
-} from './detect-builders';
-export { detectFileSystemAPI } from './detect-file-system-api';
-export { detectFramework } from './detect-framework';
-export { getProjectPaths } from './get-project-paths';
-export { DetectorFilesystem } from './detectors/filesystem';
 export { readConfigFile } from './fs/read-config-file';
 export { normalizePath } from './fs/normalize-path';
 
 export * from './should-serve';
 export * from './schemas';
 export * from './types';
 export * from './errors';
-
-/**
- * Helper function to support both `@vercel` and legacy `@now` official Runtimes.
- */
-export const isOfficialRuntime = (desired: string, name?: string): boolean => {
```

Comment:
@TooTallNate Would you agree that this function won't be used by Builders?

Reply:
I confirmed that no other repos use `isOfficialRuntime` besides clones/forks of `vercel/vercel`

https://cs.github.com/?scopeName=All+repos&scope=&q=isOfficialRuntime